'''\ Is detected as line break in triple quoted strings

```
flake8 --version
3.8.2 (flake8-broken-line: 0.2.0, flake8-bugbear: 20.1.4, flake8-colors:
0.1.0, flake8-comprehensions: 3.2.2, flake8-eradicate: 0.3.0, flake8_commas:
2.0.0, flake8_isort: 3.0.0, mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes:
2.2.0) CPython 3.6.3 on Linux
```
code to repro
```
'''\
'''
```
The `\` there is used to elminate the newline from the literal string block as noted 1/2 page down or so here (https://docs.python.org/3.6/tutorial/introduction.html#strings)

> String literals can span multiple lines. One way is using triple-quotes: """...""" or '''...'''. End of lines are automatically included in the string, but it’s possible to prevent this by adding a \ at the end of the line. The following example:


>

My minimal repro
```
% flake8 --version                                                                                                                                                                               (assets-3.6) 
3.8.2 (flake-mutable: 1.2.0, flake8-broken-line: 0.2.0, flake8-bugbear:
20.1.4, flake8-colors: 0.1.0, flake8-comprehensions: 3.2.2, flake8-eradicate:
0.3.0, flake8_commas: 2.0.0, flake8_isort: 3.0.0, flake8_quotes: 3.2.0,
flake8_strict: 0.2.1, mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0, use-
fstring-format: 1.0, use-fstring-percent: 1.0) CPython 3.6.3 on Linux
```

```
% cat file.py                                                                                                                                                                                    (assets-3.6) 
def fn():
    return """\
    """
```
```
% flake8 file.py                                                                                                                                                                                 (assets-3.6) 
file.py:2:1: N400: Found backslash that is used for line breaking
1
```

--- a/flake8_broken_line.py
+++ b/flake8_broken_line.py
@@ -13,6 +13,8 @@ pkg_name = 'flake8-broken-line'
 pkg_version: str = pkg_resources.get_distribution(pkg_name).version
 
 _INVALID_LINE_BREAK = re.compile(r'(?<!\\)\\$', re.M)
+_INVALID_MULTILINE_BACKSLASH = re.compile(r'(?<![\']{3}|[\"]{3})\\$', re.M)
+CONDITIONS = [_INVALID_LINE_BREAK, _INVALID_MULTILINE_BACKSLASH]
 _IGNORED_TOKENS = frozenset((
     tokenize.STRING,
     tokenize.COMMENT,
@@ -38,7 +40,7 @@ def check_line_breaks(
         if line_token.start[0] in reported:
             continue  # There might be several tokens on a same line.
 
-        if _INVALID_LINE_BREAK.search(line_token.line):
+        if all(condition.search(line_token.line) for condition in CONDITIONS):
             yield (*line_token.start, _N400, 'check_line_breaks')
             reported.add(line_token.start[0])
 

--- a/tests/fixtures/correct.py
+++ b/tests/fixtures/correct.py
@@ -27,3 +27,13 @@ third_multiline = """
 This is trailing backslash line break with extra spaces: \
 and next line.
 """
+
+fourth_multiline = """\
+This is a multiline block with no prepended newline
+and next line.
+"""
+
+fifth_multiline = '''\
+This is a multiline block with no prepended newline
+and next line.
+'''

